azure_functions_worker/logging.py

```diff
@@ -76,6 +76,12 @@ def enable_console_logging() -> None:
         logger.addHandler(handler)
 
 
+def enable_debug_logging_recommendation():
+    logging.info("To enable debug level logging, please refer to "
+                 "https://docs.microsoft.com/en-us/azure/azure-functions/"
+                 "functions-app-settings#python_enable_debug_logging")
```

Comment:
nit; Can you make this into an aka.ms link? Work with @shreyabatra4 to create a new link

Reply:
here it is: aka.ms/python-enable-debug-logging